webapp/components/expression/expressionEditorPopup.js

```diff
@@ -111,28 +115,32 @@ const ExpressionEditorPopup = (props) => {
 ExpressionEditorPopup.propTypes = {
   canBeConstant: PropTypes.bool, // True if expression can be a constant value like a number or a string
   expr: PropTypes.object, // AST expression
-  hideAdvanced: PropTypes.bool, // True if expression returns a boolean condition
-  isBoolean: PropTypes.bool,
+  hideAdvanced: PropTypes.bool, // True if only basic mode is allowed
+  onlyAdvanced: PropTypes.bool, // True if only advanced mode is allowed
```

Comment:
can we have those two consistent. we can pass an array for the type.
```type=[ExpressionEditor.type.basic, ExpressionEditor.type.advanced]``` by default. what do you think?

Reply:
ok, good!